src/Modal/index.jsx

```diff
@@ -20,12 +26,21 @@ class Modal extends React.Component {
 
     this.headerId = newId();
 
+    this.el = document.createElement('div');
+
+    if (!modalRoot) {
+      modalRoot = document.createElement('div');
+      modalRoot.id = modalRootId;
+      document.body.appendChild(modalRoot);
+    }
+
     this.state = {
       open: props.open,
     };
   }
 
   componentDidMount() {
+    modalRoot.appendChild(this.el);
```

Comment:
So I'm probably totally missing something here...

It looks like we're creating a `modalRoot` `div` that we're appending to document body. Then for this `modalRoot` `div` we're appending another `div` (`this.el`), which is where we're creating the portal.

Is there any reason we're not appending `this.el` to document body? In other words, what is `modalRoot` doing that `this.el` couldn't do?